sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## 11.1.0b3 (Unreleased)
 
+**Breaking Changes**
+
+- Stopped supporting `window` kwargs for `SearchIndexDocumentBatchingClient`
+- Splitted kwarg `hook` into `new_callback`, `progress_callback`, `error_callback`, `remove_callback` for `SearchIndexDocumentBatchingClient`
```

Comment:
We continue to use the suffix `_hook` or the prefix `on_` to represent callbacks in other SDKs - are we able to align?
e.g.
- [storage blobs](https://github.com/Azure/azure-sdk-for-python/blob/feature/storage-stg74/sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py#L732) `on_error`
- [event hubs](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/samples/sync_samples/recv_for_period.py#L62) `on_error`/`on_partition_initialize` etc
- In Datalake we use the `_hook` suffix when the callback handles a few different operations (e.g. not just errors).

Reply:
Update changelog to reflect new `on_` prefix to callbacks